chan.go

```diff
@@ -57,7 +61,15 @@ func (r *chanResponse) Error() *cmdkit.Error {
 		return nil
```

Comment:
Can we get rid of both of these nil tests? I'd rathe panic if something is wrong (and we'll panic in the second case anyways).

Reply:
well the `r == nil` check obviously doesn't help because we already access `r.wait` before. But what's wrong with the `r.err == nil` check?

Reply:
We call `r.err.Error()` below, so `r.err` better not be nil.

Reply:
Sorry, I was refering to both of the `r == nil` checks (in this function and the one above). Unfortunately, git hides that part of the patch.